tests/h/presenters/annotation_searchindex_test.py

```diff
@@ -57,120 +54,49 @@ def test_asdict(self, DocumentSearchIndexPresenter, pyramid_request, thread_ids)
             ],
             "document": {"foo": "bar"},
             "references": ["referenced-id-1", "referenced-id-2"],
-            "thread_ids": thread_ids,
+            "thread_ids": ["thread-id-1", "thread-id-2"],
             "hidden": False,
         }
 
-    def test_it_copies_target_uri_normalized_to_target_scope(self, pyramid_request):
-        annotation = mock.MagicMock(
-            userid="acct:luke@hypothes.is",
-            target_uri_normalized="http://example.com/normalized",
-            extra={},
-        )
-
-        annotation_dict = AnnotationSearchIndexPresenter(
-            annotation, pyramid_request
-        ).asdict()
-
-        assert annotation_dict["target"][0]["scope"] == [
-            "http://example.com/normalized"
-        ]
-
-    def test_it_marks_annotation_hidden_when_it_and_all_children_are_moderated(
-        self, pyramid_request, moderation_service, thread_ids
-    ):
-        annotation = mock.MagicMock(
-            userid="acct:luke@hypothes.is", thread_ids=thread_ids
-        )
-
-        moderation_service.all_hidden.return_value = [annotation.id] + thread_ids
-
-        annotation_dict = AnnotationSearchIndexPresenter(
-            annotation, pyramid_request
-        ).asdict()
-
-        assert annotation_dict["hidden"] is True
-
-    def test_it_does_not_mark_annotation_hidden_when_it_is_not_moderated(
-        self, pyramid_request, moderation_service, thread_ids
-    ):
-        annotation = mock.MagicMock(
-            userid="acct:luke@hypothes.is", thread_ids=thread_ids
-        )
-
-        moderation_service.all_hidden.return_value = thread_ids
-
-        annotation_dict = AnnotationSearchIndexPresenter(
-            annotation, pyramid_request
-        ).asdict()
-
-        assert annotation_dict["hidden"] is False
```

Comment:
I've boiled these four different tests into a parameterised truth table basically, which I think makes it both much shorter, and much clearer what we are actually testing.

If _both_ us and all our replies are moderated, we are hidden.